x-pack/plugins/security_solution/public/resolver/test_utilities/simulator/index.tsx

```diff
@@ -220,6 +220,33 @@ export class Simulator {
     );
   }
 
+  /**
+   * For a render of the Enzyme wrapper (may help update if other concerns have effected the render). May be useful for debugging.
+   */
+  public updateWrapper() {
```

Comment:
can you remove this for now (its unused)

Reply:
https://github.com/elastic/kibana/pull/74411/commits/a48a10a6244e3ea35eda3de095fddb427e645c91#diff-d5aefb42e7a51c722db06ebb62bc1a47L223